File: leoz-common/src/test/java/org/deku/leoz/rest/v1/StationServiceTest.java
package org.deku.leoz.rest.v1;

import org.deku.leoz.rest.WebserviceTest;
import org.deku.leoz.rest.entities.internal.v1.Station;
import org.deku.leoz.rest.services.internal.v1.StationService;
import org.junit.Ignore;
import org.junit.Test;

/**
 * Created by masc on 17.09.14.
 */
public class StationServiceTest extends WebserviceTest {
    @Test
    public void testGet() throws Exception {

    }

    @Test
    @Ignore
    public void testGetPerformance() throws Exception {
        StationService ts = this.getService(StationService.class);

        for (int i = 0; i < 20; i++) {
            Station[] stations = ts.get();

            for (Station d : stations) {
                System.out.println(String.format("%d %s", i, d.toString()));
            }
        }
    }
}

File: leoz-core/src/test/java/org/deku/leoz/rest/v1/StationServiceTest.kt
package org.deku.leoz.rest.v1

import org.deku.leoz.rest.WebserviceTest
import org.deku.leoz.rest.entity.internal.v1.Station
import org.deku.leoz.rest.service.internal.v1.StationService
import org.junit.Ignore
import org.junit.Test
import org.slf4j.LoggerFactory

/**
 * Created by masc on 17.09.14.
 */
class StationServiceTest : WebserviceTest() {
    private val log = LoggerFactory.getLogger(this.javaClass)
    @Test
    @Throws(Exception::class)
    fun testGet() {
        val ts = this.getService(StationService::class.java)
        val stations = ts.get()
        log.info("Received ${stations.count()} stations")
    }
}
